packages/main/src/Icon.js

```diff
@@ -30,6 +30,31 @@ const metadata = {
 		src: {
 			type: String,
 		},
+
+		/**
+		 * Defines the text alternative of the <code>ui5-icon</code>.
+		 * If not provided a default text alternative will be set, if present.
+		 * <br><br>
+		 * <b>Note:</b> Every icon should have a text alternative in order to
+		 * calculate its accessible name.
+		 *
+		 * @type {string}
+		 * @public
+		 */
+		accessibleName: {
+			type: String,
+		},
+
+		/**
+		 * Defines whether the <code>ui5-icon</code> should have a tooltip.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		showIconTooltip: {
```

Comment:
IMO just showTooltip (Icon is in the name of the element already)

Reply:
I was thinking a lot about this name! Thanks, I have changed it :)